Crypto/css/style.css

```diff
@@ -0,0 +1,1029 @@
+    margin-top: 95px;
+}
+
+.section6-item-block {
+    width: 33.3%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    border: 1px solid #C23893;
+}
+
+.section6-item-block:nth-child(1),
+.section6-item-block:nth-child(2),
+.section6-item-block:nth-child(3) {
+    border-top: none;
+}
+
+.section6-item-block:nth-child(1),
+.section6-item-block:nth-child(4) {
+    border-left: none;
+}
+
+.section6-item-block:nth-child(3),
+.section6-item-block:nth-child(6) {
+    border-right: none;
+}
+
+.section6-item-block:nth-child(n+4) {
+    border-bottom: none;
+}
+
+.section6-item-block:nth-child(n+4) {
+    padding-top: 58px;
+}
+
+.section6-item-block img {
+    width: 40px;
+    height: 40px;
+}
+
+.section6-item-block h1 {
+    font-family: var(--font-primary);
+    font-style: normal;
+    font-weight: 700;
+    font-size: 21px;
+    line-height: 28px;
+    text-align: center;
+    color: #FFFFFF;
+    margin-top: 47px;
+}
+
+.section6-item-block p {
+    display: block;
+    max-width: 330px;
+    font-family: var(--font-primary);
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 26px;
+    text-align: center;
+    color: #767799;
+    margin-top: 22px;
+    margin-bottom: 64px;
+}
+
+/* Section 7 */
+
+.section7-item-container {
+    display: flex;
+    justify-content: space-between;
+    max-width: 1240px;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+.section7-left-item {
+    margin-top: 165px;
+    display: block;
+    position: relative;
+}
+
+.section7-left-item-elips img {
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    display: block;
+    position: absolute;
+    z-index: -1;
+    width: auto;
+    height: auto;
+}
+
+.section7-left-item-iphone img {
+    max-width: 100%;
+    height: auto;
+}
+
+.section7-right-item {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    max-width: 581px;
+    margin-top: 245px;
+}
+
+.section7-right-item h1 {
+    font-family: var(--font-primary);
+    font-style: normal;
+    font-weight: 700;
+    font-size: 25px;
+    line-height: 104.5%;
+    color: #962F79;
+}
+
+.section7-right-item h2 {
+    font-family: var(--font-primary);
+    font-style: normal;
+    font-weight: 500;
+    font-size: 60px;
+    line-height: 104.5%;
+    color: #FFFFFF;
+    margin-top: 19px;
+}
+
+.section7-right-item p {
+    font-family: var(--font-secondary);
+    font-style: normal;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 168.5%;
+    color: #767799;
+    margin-top: 37px;
+}
+
+.section7-right-item a {
+    display: inline-block;
+    border: 1px solid #FFFFFF;
+    text-decoration: none;
+    padding: 20px 44.5px 20px 44.5px;
+    font-family: var(--font-secondary);
+    font-style: normal;
+    font-weight: 600;
+    font-size: 21px;
+    line-height: 25px;
+    text-transform: capitalize;
+    color: #FFFFFF;
+    margin-top: 54px;
+    max-width: fit-content;
```

Comment:
not sure if you need fit-content, based on your code it will grow based on the content inside.